spec/features/users/05_user_can_view_story_link_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature 'user creates a stoy link', %{
+  As an authenticated user
+  Link to another user's story
+  So new users can view my story link on that user's story show page
+} do
+
+  # Acceptance Criteria:
+  # [] User2 story appears on User1 story show page
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:story) { FactoryGirl.create(:story,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.